.github/workflows/pr.yml

```
@@ -80,89 +66,42 @@ jobs:
           yarn bundle+esbuild
         shell: bash
         working-directory: packages/test-app
-  build-android:
+  build-android-expo-app:
     name: "Build Android"
-    runs-on: ubuntu-latest
-    steps:
-      - name: Set up JDK
-        uses: actions/setup-java@v3.3.0
-        with:
-          distribution: temurin
-          java-version: 11
-      - name: Set up Node 14
-        uses: actions/setup-node@v3.3.0
-        with:
-          node-version: 14
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: Cache /.yarn-offline-mirror
-        uses: actions/cache@v3
-        with:
-          path: .yarn-offline-mirror
-          key: ${{ runner.os }}-${{ hashFiles('yarn.lock') }}-7
-      - name: Install npm dependencies
-        run: yarn ci
-        env:
-          CI_SKIP_GO: 1
-      - name: Build Android app
-        run: |
-          ./gradlew clean build
-        working-directory: packages/test-app/android
-  build-ios:
+    uses: ./.github/workflows/build-android.yml
+    with:
+      package: expo-app
+  build-android-test-app:
+    name: "Build Android"
+    uses: ./.github/workflows/build-android.yml
+    with:
+      package: test-app
+  build-ios-expo-app:
     name: "Build iOS"
```

We should use a different name so we can distinguish them in the sidebar.
```suggestion
    name: "Build Expo iOS"
```